content/en/docs/Introduction/admission-controllers.md

```diff
@@ -0,0 +1,90 @@
+
+Following is the list of use cases of admission controllers:
+
+- **Security:** Admission controllers can increase security by mandating a reasonable security baseline across an entire namespace or cluster. For Example, PodSecurityPolicy is a built-in admission controller that can be used for disallowing containers from running as root or making sure the container’s root filesystem is always mounted read-only (for example). Furthermore, you can use webhook-based admission controllers to achieve use cases like allowing pulling of images only from specific registries known to the enterprise. Or Rejecting deployments that do not meet security standards.
+
+- **Governance:** Using Admission controllers you can enforce the adherence to certain practices like having good labels, annotations, resource limits, or other settings. For example, you can enforce label validation on different objects to ensure proper labels are being used, such as every object being assigned to a team or project, or every deployment specifying an app label. In another example, You can enforce annotations automatically being added to objects, such as attributing the correct cost center for a “dev” deployment resource.
+
+- **Configuration management:** Admission controllers allow you to validate the configuration of the objects running in the cluster and prevent any obvious misconfigurations from hitting your cluster. Admission controllers can be useful in detecting and fixing images deployed without semantic tags, such as by automatically adding resource limits or validating resource limits, ensuring reasonable labels are added to pods, or ensuring image references used in production deployments are not using the latest tags, or tags with a -dev suffix.
+
+In this way, admission controllers help in ensuring that applications stay in compliance within an ever-changing landscape of controls.
+
+## Using Admission Controllers
+
+Admission Controllers can be added by the administrator typically at the time of cluster creating by passing a flag to the `kube-apiserver`.
+
+```sh
+kube-apiserver --enable-admision-plugins=NamespaceLifecycle
+```
+
+Kubernetes recommends the following admission controllers to be enabled by default.
+
+```sh
+--enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,Priority,ResourceQuota,PodSecurityPolicy
+```
+
+ You can find which admission plugins are enabled by:
+
+```sh
+kube-apiserver -h | grep enable-admission-plugins
+```
+
+disable-admission-plugins flag is used to disable a comma-delimited list of admission controllers. This list may also include admission controllers enabled by default.
+
+```sh
+kube-apiserver --disable-admission-plugins=PodNodeSelector,AlwaysDeny
+```
+
+## Admission Controllers (Kubernetes v1.28)
```

Comment:
This entire section is unnecessary for a 101-level introduction to admission controllers. It's fine to list a couple and explain what they are and what they do and then provide a link to the docs, but this document isn't about being a reference guide.